vulcan/lib/server/workflows/bar-plot.cwl

```diff
@@ -9,18 +9,13 @@ inputs:
   1_Data_Source_magma_query__user_columns:
     type: string
     label: "user_columns input"
-    doc: "Optional array or comma-separated set of *quoted* strings for renaming columns of the returned data.  *Coming Soon* Suggestion: Use the Query page of Timur to build the proper dataframe, and then simply copy over the 'user_columns'-chunk presented there!"
+    doc: "Optional array or comma-separated set of *quoted* strings for renaming columns of the returned data.  Suggestion: Use the Query page of Timur to build the proper dataframe, and then simply copy over the 'user_columns'-chunk presented there!"
```

Comment:
nit / question: what is an "array or comma-separated set"? You're saying the input will take either:

`"bar", "foo", "bim"` (comma-separated set)

or ...? What is an example of array?

Reply:
oooh good point.  It was meant to get at that you can give either:
- `"bar", "foo", "bim"`
- `["bar", "foo", "bim"]`

Lemme update to make that more clear.

Reply:
I removed the array term.  I have a sense that it doesn't matter to say at all...?  Do you agree?

When the user copies from Query, they'll have the brackets, which is fine.  When a user creates the input here manually, they don't need to add the [] because it will be added in the script when needed, prior to the query call.